acceptance/tests/aix/aix_package_provider.rb

```diff
@@ -20,7 +20,7 @@ def assert_package_version(package, expected_version)
 
 teardown do
   on hosts, "rm -rf #{dir}"
-  on hosts, puppet('resource', 'package', "'#{package}' ensure=absent")
+  on hosts, "installp -u #{package}"
```

Comment:
This change only affects the package installed by the test ({{sudo.rte}}) and not puppet-agent packages. I am not sure that we want this teardown to extend beyond the scope of the packages the test installs.